src/Endatix.Api/Infrastructure/Cors/EndpointsCorsConfigurator.cs

```diff
@@ -0,0 +1,163 @@
+using Endatix.Framework.Hosting;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Endatix.Api.Infrastructure.Cors;
+
+public class EndpointsCorsConfigurator(IOptions<CorsSettings> corsSettings, ILogger<EndpointsCorsConfigurator> logger, IAppEnvironment appEnvironment, IWildcardSearcher wildcardSearcher) : IConfigureOptions<CorsOptions>
+{
+    public const string ALLOW_ALL_POLICY_NAME = "AllowAll";
+
+    public const string DISALLOW_ALL_POLICY_NAME = "DisallowAll";
+
+    private readonly string[] _no_input_params = [];

```

Comment:
I think camelCase  is more appropriate here, like `_noInputParams`.
Considering the leading _, I think we have both private private fields (with and without leading _) in the code base. It will be good to decide which convention we will follow but this is not for the PR definitely.